test/e2e_rootless/rootlessmode.go

```diff
@@ -0,0 +1,45 @@
+package e2erootlessmode
+
+import (
+	"bytes"
+	"strings"
+
+	"github.com/loft-sh/vcluster/test/framework"
+	"github.com/onsi/ginkgo"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/tools/remotecommand"
+)
+
+var _ = ginkgo.Describe("Rootless mode", func() {
+	f := framework.DefaultFramework
+	ginkgo.It("run vcluster in rootless mode", func() {
+		pods, err := f.HostClient.CoreV1().Pods(f.VclusterNamespace).List(f.Context, metav1.ListOptions{
+			LabelSelector: "app=vcluster",
+		})
+		framework.ExpectNoError(err)
+		vclusterPod := pods.Items[0].Name
+		req := f.HostClient.CoreV1().RESTClient().Post().Resource("pods").Name(vclusterPod).Namespace(f.VclusterNamespace).SubResource("exec")
+		cmd := []string{
+			"/bin/sh",
+			"-c",
+			"id -u",
+		}
+		req.VersionedParams(
+			&v1.PodExecOptions{
+				Container: "syncer",
+				Command:   cmd,
+				Stdout:    true,
+			},
+			clientgoscheme.ParameterCodec,
+		)
+		exec, err := remotecommand.NewSPDYExecutor(f.HostConfig, "POST", req.URL())
+		framework.ExpectNoError(err)
+		var stdout bytes.Buffer
+		err = exec.Stream(remotecommand.StreamOptions{Stdout: &stdout})
```

Comment:
btw, we have util to make exec easier - https://github.com/loft-sh/vcluster/blob/main/pkg/util/podhelper/exec.go#L152-L153